Description: add linker flags for check
Origin: vendor
Forwarded: no
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2013-06-26

--- a/tests/Makefile
+++ b/tests/Makefile
@@ -6,7 +6,7 @@
 OS = `uname | tr "a-z" "A-Z"`
 
 CHECK_PATH = /usr/local
-LDFLAGS = -L$(CHECK_PATH)/lib -lcheck `../src/osflags link`
+LDFLAGS = -L$(CHECK_PATH)/lib -lcheck `pkg-config --cflags --libs check` `../src/osflags link`
 CFLAGS += -D$(OS) -I../src -I$(CHECK_PATH)/include -pedantic `../src/osflags cflags` $(CPPFLAGS)
 
 all: $(TEST)
