From: gregor herrmann <gregoa@debian.org>
Date: Wed, 2 Apr 2014 13:07:40 +0100
Subject: verbose-build

---
 tests/Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tests/Makefile b/tests/Makefile
index 2dc1168..d4433ba 100644
--- a/tests/Makefile
+++ b/tests/Makefile
@@ -14,11 +14,11 @@ all: $(TEST)
 
 $(TEST): $(OBJS) $(SRCOBJS)
 	@echo LD $(TEST)
-	@$(CC) -o $@ $(SRCOBJS) $(OBJS) $(LDFLAGS) 
+	$(CC) -o $@ $(SRCOBJS) $(OBJS) $(LDFLAGS)
 
 .c.o: 
 	@echo CC $<
-	@$(CC) $(CFLAGS) -c $<
+	$(CC) $(CFLAGS) -c $<
 
 clean:
 	@echo "Cleaning tests/"
